pkg/reconciler/revision/resources/queue_test.go

```diff
@@ -101,6 +101,36 @@ func TestMakeQueueContainer(t *testing.T) {
 				"CONTAINER_CONCURRENCY": "1",
 			})
 		}),
+	}, {
+		name: "custom readiness probe port",
+		rev: revision("bar", "foo",
+			withContainers([]corev1.Container{{
+				Name: servingContainerName,
+				ReadinessProbe: &corev1.Probe{
+					Handler: corev1.Handler{
+						TCPSocket: &corev1.TCPSocketAction{
+							Host: "127.0.0.1",
+							Port: intstr.FromInt(8087),
```

Comment:
Shouldn't this affect the `SERVING_READINESS_PROBE` ? but I don't see it in `want: queueContainer(...)` ?

Reply:
the `SERVING_READINESS_PROBE` is computed in the test using the func `probeJSON` in the same file which got modified as you can see on line 929, so its gets the correct value of the port (8087) in the comparison without changing the want section

Reply:
https://github.com/knative/serving/blob/43f84e85acfa5e0a48a2cbf223ed44cca5672f45/pkg/reconciler/revision/resources/queue_test.go#L370-L373

Reply:
oh - I see - that's confusing - thanks for clarifying